opm/simulators/aquifers/AquiferFetkovich.hpp

```diff
@@ -64,8 +64,8 @@ class AquiferFetkovich : public AquiferInterface<TypeTag>
     {
         for (const auto& q : this->Qai_) {
             this->W_flux_ += q * this->ebos_simulator_.timeStepSize();
-            aquifer_pressure_ = aquiferPressure();
         }
+        aquifer_pressure_ = aquiferPressure();
```

Comment:
Is it more readable if this->W_flux_ is communicated here and not in aquiferPressure()

Reply:
W_flux_ is an Evaluation so I cannot easily collect the global values. Thus I see no way to move the reduction.